src/GameLoop.tsx

```diff
@@ -554,3 +583,21 @@ export function GameLoop() {
         </div>
     );
 }
+
+export const CountdownOverlay = React.memo(
+    ({ isVisible, countdownNum }: { isVisible: boolean, countdownNum: number }) => {
+        const divStyle = {
+            visibility: isVisible ? "visible" as const : "hidden" as const,
+            position: 'absolute',
+            top: '35%',
+            left: '50%',
+            transform: 'translate(-50%, -50%)',
+            zIndex: 2,
+            color: 'red',
+            fontSize: '200%',
```

Comment:
```suggestion
            position: "absolute",
            top: "35%",
            left: "50%",
            transform: "translate(-50%, -50%)",
            zIndex: 2,
            color: "red",
            fontSize: "200%",
```

We use double quotes everywhere else.